hicommonwealth/commonwealth
# 7030 stake modal icon

## Link to Issue
Closes: #7030

## Description of Changes
- Added icons for the chains we use for stake
- Built some logic to render the correct string / icons

## "How We Fixed It"

- Added `denomination` prop to `ManageCommunityStakeModal` 
- Added logic to `useCommunityStake` to format the current chain correctly so it can be passed around
- Using object literals, we return the correct icon based on the staking chain 

## Test Plan
- Create a community and enable community stake
- Open the staking modal (buy/sell)
- Ensure the denomination is correct
### To test other denominations
- In `useCommunityStake` line: 31, hard set the variable `activeChainId` to the three different options: "Ethereum", "Blast" or "Base"
- Observe and ensure the result matches the [figma](https://www.figma.com/file/omZZrndm6Phbk1rOU4IKhS/Stake-V1-Fast-Follows?type=design&node-id=458-3752&mode=design&t=0NjAm0EvRfckBv9u-0)
